lib/ReactViews/Map/Panels/SettingPanel.tsx

```diff
@@ -128,6 +128,26 @@ class SettingPanel extends React.Component<PropTypes> {
     this.props.terria.currentViewer.notifyRepaintRequired();
   }
 
+  onBaseMaximumScreenSpaceErrorChange(bmsse: number) {
+    this.props.terria.setBaseMaximumScreenSpaceError(bmsse);
+    if (bmsse) {
```

Comment:
When would this condition be false? Can bmsse be 0? If it's a 0 comparison I think it's better to be explicit `bmsse !== 0`.